includes/admin/tools/import/class-give-import-donations.php

```diff
@@ -820,6 +828,7 @@ public function save() {
 						'delete_csv'    => empty( $_POST['delete_csv'] ) ?
 							'1' :
 							( give_is_setting_enabled( give_clean( $_POST['delete_csv'] ) ) ? '1' : '0' ),
+						'per_page'      => absint( isset( $_POST['per_page'] ) ? $_POST['per_page'] : self::$per_page ),
```

Comment:
@raftaar1191 Can you explain `absint( isset( $_POST['per_page'] )`

Reply:
Update the code 

Thanks @ravinderk for pointing this out to me.